package.json

```diff
@@ -64,6 +64,7 @@
     "node-sass": "^4.12.0",
     "pg": "^7.18.2",
     "prop-types": "^15.7.2",
+    "qrcode.react": "^1.0.0",
```

Comment:
Sorry, I didn't want you to add the package in this branch. It was just to clear up the build error. The package is being added in the other pr, right?

Reply:
You mean this PR https://github.com/HackYourFuture-CPH/fp-class12/pull/105, right?

Reply:
yes